github/ipfs.yml

```
@@ -5686,6 +5686,18 @@ teams:
         - ntninja
         - whereswaldon
     privacy: closed
+  ipfs-devrel:
```

If we go with just devrel, then we could mention it with @ipfs/devrel
```suggestion
 devrel:
```

Completely up to you, let me know :)

---

much better :)